rclpy_examples/talker_py.py

```diff
@@ -0,0 +1,29 @@
+import sys
+
+import rclpy
+from rclpy.qos import qos_profile_default
+from std_msgs.msg import String
+
+
+def main(args):
+    rclpy.init(args)
+
+    node = rclpy.create_node("talker")
+
+    chatter_pub = node.create_publisher(String, "chatter", qos_profile_default)
+
+    loop_rate = rclpy.WallRate(2)
+
+    msg = String()
+
+    i = 1
+    while rclpy.ok():
+      msg.data = "Hello World: " + i
```

Comment:
This will not work I think, should give an error like `No '+' operator between types 'str' and 'int'`. You'll need to do a formatted string `msg.data = "Hello World: {0}".format(i)`.

Btw, in Python3.6 we can use string interpolation! Like this: `msg.data = f"Hello World: {i}"` I'm so excited: https://www.python.org/dev/peps/pep-0498/